src/os/bluestore/BlueStore.cc

```diff
@@ -5408,7 +5408,16 @@ void BlueStore::_txc_add_transaction(TransContext *txc, Transaction *t)
       break;
 
     case Transaction::OP_MERGE_DELETE:
-      assert(0 == "not implemented");
+      {
+	const ghobject_t& noid = i.get_oid(op->dest_oid);
+	OnodeRef& no = ovec[op->dest_oid];
+	if (!no) {
+	  no = c->get_onode(noid, true);
```

Comment:
Sam,

If base object is deleted before move transaction by some one, do we continue with move api by creating a new base object or would we like to return without doing anything/or deleting the temp obj?
here, we are creating a new base obj and continuing with move api.

Also, when the move api is called the first time, has the ECbackend code created base object ?
or will move api create it after finding out that its not present?